Controller/RciamStatsViewerServicesController.php

```diff
@@ -96,7 +116,7 @@ public function getdataforuserstiles()
   {
     $this->log(__METHOD__ . '::@', LOG_DEBUG);
     // We accept only Ajax Requests
-    if(!$this->request->is('Ajax')) {
+    if (!$this->request->is('Ajax')) {
```

Comment:
Undo the white space import change here(https://spaces.at.internet2.edu/display/COmanage/COmanage+Coding+Style#COmanageCodingStyle-Alignment)
Also try to follow the same pattern in all other occasions.

Reply:
ok done.